postgresql/hooks/init

```diff
@@ -1,13 +1,13 @@
-#!{{pkgPathFor "core/bash"}}/bin/bash
+#!/bin/sh
+#
```

Comment:
If we drop the `core/bash` runtime requirement, we'll want to make sure that all hooks and the sourced `functions.sh` are bourne shell compatible.

At the moment in `init` I'm seeing a `[[`-style test that would need to be converted to a `[` style and IRRC we'd need to change `source {{pkg.svc_config_path}}/functions.sh` to `. {{pkg.svc_config_path}}/functions.sh` as well.